src/Hwloc.jl

```diff
@@ -207,7 +207,7 @@ end
 type Object
     type_::Symbol
     os_index::Int
-    name::Compat.UTF8String
+    name::Compat.String
```

Comment:
Hi @eschnett, I made the change based on the warning from Compat that suggested replacing UTF8String with String. The Travis CI build shows that the change works in Julia v0.5. By maintainer you mean someone that can merge PRs? I would be interested, but I will definitely need someone else's approval if I make any major change in the code.